src/main/java/com/samhap/kokomen/interview/external/dto/response/InterviewSummaryResponses.java

```diff
@@ -16,10 +17,12 @@ public static InterviewSummaryResponses createOfOtherMemberForAuthorized(
             Long totalMemberCount,
             Long intervieweeRank,
             List<Interview> interviews,
-            Set<Long> likedInterviewIds
+            Set<Long> likedInterviewIds,
+            Map<Long, Long> viewCounts
     ) {
```

Comment:
```suggestion
            Map<Interview, Long> intervewAndViewCounts
    ) {
```

위와 같이 바꾸면 `List<Interview> interviews` 를 받을 필요가 없게 되는데 어떤 방식이 더 좋다고 생각하실까요?

Reply:
조회수를 받다가 안 받게 되면 코드를 다시 돌리기 귀찮아 질 것 같다는 생각이 들긴 하는데요.
사실 인터뷰 목록을 조회할 때 갑자기 조회수를 안 받겠다는 요구사항이 들어올 가능성은 없을 것 같네요.

그런 관점에서 파라미터 수 줄이는 용도로, 말씀하신대로 바꾸셔도 괜찮아 보입니다!